inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_b_fs_yx_fsv4_int8.cl

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "include/common.cl"
+#include "include/data_types.cl"
+#include "include/fetch.cl"
+#include "include/imad.cl"
+
+#if QUANTIZATION_TERM
+#    define ACCUMULATOR_TYPE int
+#    define TO_ACCUMULATOR_TYPE(x) convert_int(x)
+#    define ACTIVATION_TYPE float
+#    define TO_ACTIVATION_TYPE(x) convert_float(x)
+#else
+#    define ACCUMULATOR_TYPE INPUT0_TYPE
+#    define TO_ACCUMULATOR_TYPE(x) TO_INPUT0_TYPE(x)
+#    define ACTIVATION_TYPE INPUT0_TYPE
+#    define TO_ACTIVATION_TYPE(x) TO_INPUT0_TYPE(x)
+#endif
+
+__attribute__((intel_reqd_sub_group_size(SUB_GROUP_SIZE)))
+__attribute__((reqd_work_group_size(1, 1, SUB_GROUP_SIZE)))
+KERNEL(convolution_gpu_b_fs_yx_fsv4_int8)(
+    const __global float* input,
```

Comment:
float -> uint or define INPUT0_PACKED_TYPE and use it

Reply:
Done